<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gams::studio::option::GamsConfigEditor</class>
 <widget class="QWidget" name="gams::studio::option::GamsConfigEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="GamsCfgTabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="gams::studio::option::ParamConfigEditor" name="gamsParamTab">
      <attribute name="title">
       <string>Command Line Parameters</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="envVariableTab">
      <attribute name="title">
       <string>Environment Variables</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gams::studio::option::ParamConfigEditor</class>
   <extends>QWidget</extends>
   <header>option/paramconfigeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
